[2020-01-11 15:45:52,448] - My_Logger.py] - INFO: 用例失败,失败原因 :: 500 != 0,
接口实际code:500
[2020-01-11 15:45:52,549] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 15:46:28,914] - My_Logger.py] - INFO: 用例失败,失败原因 :: 500 != 0,
接口实际code:500
[2020-01-11 15:46:29,001] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 15:50:09,170] - My_Logger.py] - INFO: 用例失败,失败原因 :: 500 != 0,
接口实际code:500
[2020-01-11 15:50:09,267] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 15:51:18,542] - My_Logger.py] - INFO: 用例失败,失败原因 :: 500 != 0,
接口实际code:500
[2020-01-11 15:51:18,641] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 16:08:16,474] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 16:08:16,581] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 16:09:22,841] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 16:09:22,939] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 16:10:39,406] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 16:10:39,504] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 16:15:23,775] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 16:15:23,868] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 16:20:53,458] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 16:20:53,549] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 16:22:59,433] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 16:22:59,553] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 16:25:54,583] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-11 16:25:54,684] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:422
[2020-01-15 16:59:43,196] - My_Logger.py] - INFO: 断言错误: :: 0 != 0
[2020-01-15 17:00:02,824] - My_Logger.py] - INFO: 断言错误: :: 0 != 422
[2020-01-15 17:00:02,858] - My_Logger.py] - INFO: 断言错误: :: 422 != 401
[2020-01-15 17:00:43,001] - My_Logger.py] - INFO: 断言错误: :: 0 != 0
[2020-01-15 17:00:43,059] - My_Logger.py] - INFO: 断言错误: :: 422 != 401
[2020-01-15 17:01:39,141] - My_Logger.py] - INFO: 断言错误: :: 0 != 422
[2020-01-15 17:01:39,178] - My_Logger.py] - INFO: 断言错误: :: 422 != 401
[2020-01-15 17:14:14,494] - My_Logger.py] - INFO: 断言错误: :: 0 != 0
[2020-01-15 17:14:14,542] - My_Logger.py] - INFO: 断言错误: :: 422 != 401
[2020-01-15 17:18:18,057] - My_Logger.py] - INFO: 断言错误: :: 422 != 401
[2020-01-15 17:22:44,737] - My_Logger.py] - INFO: 断言错误: :: 0 != 422
[2020-01-15 17:29:42,496] - My_Logger.py] - INFO: 断言错误: :: 0 != 422
[2020-01-15 17:53:00,550] - My_Logger.py] - INFO: 断言错误: :: 0 != 422
[2020-01-15 17:53:16,314] - My_Logger.py] - INFO: 断言错误: :: 0 != 422
[2020-01-15 17:54:56,763] - My_Logger.py] - INFO: 断言错误: :: 0 != 422
[2020-01-16 19:02:58,764] - My_Logger.py] - INFO: 用例失败,失败原因 :: 0 != 401,
接口实际code:401
[2020-01-16 19:02:58,838] - My_Logger.py] - INFO: 用例失败,失败原因 :: 0 != 401,
接口实际code:401
[2020-01-16 19:02:58,919] - My_Logger.py] - INFO: 用例失败,失败原因 :: 0 != 401,
接口实际code:401
[2020-01-16 19:02:58,996] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 401,
接口实际code:401
[2020-01-16 19:03:45,176] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:05:22,546] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:05:35,273] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:05:37,876] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:05:40,594] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:05:43,986] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:05:47,124] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:05:50,010] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:05:53,115] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:05:56,340] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:05:59,112] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:02,045] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:05,015] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:07,884] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:11,241] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:14,380] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:17,016] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:19,844] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:22,615] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:25,359] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:28,218] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:31,178] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:33,861] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:36,823] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:39,564] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:42,574] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:45,555] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:48,882] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:51,993] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:54,652] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-01-16 19:06:57,328] - My_Logger.py] - INFO: 用例失败,失败原因 :: 422 != 0,
接口实际code:0
[2020-03-05 14:02:18,656] - My_Logger.py] - INFO: 用例失败,失败原因 :: 0 != 401,
接口实际code:401
